### PR TITLE
fix(docs): document base-path in api.yml

### PR DESCRIPTION
**Pull request by dsinghvi**
This PR documents the `base-path` config in `api.yml`

### CODE REVIEW

File: fern/docs.yml [hunk 1]
```diff
@@ -379,7 +379,7 @@ colors:
     light: "#1DA32B"
   background:
     dark: "#081008"
-    light: "#FBFFFA"
+    light: "#FFFFFF"
```
Comment (armandobelardo) [inline fern/docs.yml:382]: WOW

File: fern/pages/api-definition/fern-definition/api-yml/environments.mdx [hunk 2]
```diff
@@ -65,6 +65,17 @@ default-environment: Production
...
 
+<Note> By providing a default environment, the generated SDK will be setup to hit the default.</Note>
+
+## Base Path
+If you would like all of your endpoints to be prefixed with a path, use `base-path`. 
+
+In the example below, every endpoint is prefixed with a `/v1`: 
+```yaml api.yml
+name: api
+base-url: /v1
```
Comment (RohinBhargava) [inline fern/pages/api-definition/fern-definition/api-yml/environments.mdx:76]: nit: is this supposed to be `base-path`?

Comment (dsinghvi) [reply fern/pages/api-definition/fern-definition/api-yml/environments.mdx:76]: thank you

File: fern/pages/api-definition/fern-definition/api-yml/environments.mdx [hunk 3]
```diff
@@ -65,6 +65,17 @@ default-environment: Production
 ```
 </CodeBlock>
 
+<Note> By providing a default environment, the generated SDK will be setup to hit the default.</Note>
```
Comment (armandobelardo) [inline fern/pages/api-definition/fern-definition/api-yml/environments.mdx:68]:
```suggestion
<Note> By providing a default environment, the generated SDK will be setup to hit that URL by out-of-the-box.</Note>
```

Just to sound a little less redundant

**Review by armandobelardo — APPROVED** [review]

File: fern/pages/api-definition/fern-definition/api-yml/environments.mdx (hunk 3 above)
Comment (dsinghvi) [reply fern/pages/api-definition/fern-definition/api-yml/environments.mdx]: thank you!

**Pull request merged**